src/lib/pipeline_configuration.py

```diff
@@ -48,7 +50,8 @@ class PipelineConfiguration(object):
                    icr_filename="capyei_valuable.csv",
                    analysis_file_key="capyei_valuable_",
                    cleaner=None,
-                   code_scheme=None),
+                   binary_code_scheme=CodeSchemes.VALUEABLE,
```

Comment:
Don't you also need to set the binary_coded_field for the import to work?

Reply:
Turns out we won't be using multiple/binary schemes, removing.